{"name": "Логин пользователя: логин с неверным логином и паролем", "status": "broken", "statusDetails": {"message": "KeyError: 'accessToken'", "trace": "self = <tests.test_user_login.TestUserLogin object at 0x00000145D2BD3380>, field = 'email'\n\n    @allure.title('Логин пользователя: логин с неверным логином и паролем')\n    @pytest.mark.parametrize('field',\n                             [\n                                 pytest.param('email', id='тест с email'),\n                                 pytest.param('password', id='тест с password')\n                             ]\n                             )\n    def test_user_login_with_wrong_params(self, field):\n        user_body = UserData.generate_fake_user_data()\n        user = Auth.create_user(user_body)\n        user_body_v2 = user_body.copy()\n        user_body_v2[field] = user_body[field] + 'SOMETHING_WRONG'\n        login = Auth.user_login(user_body_v2)\n    \n>       Auth.delete_user(user)\n\ntests\\test_user_login.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nuser_response = <Response [403]>\n\n    @staticmethod\n    @allure.step('Отправить запрос на удаление пользователя')\n    def delete_user(user_response):\n>       access_token = user_response.json()[\"accessToken\"]\nE       KeyError: 'accessToken'\n\nmodules\\auth.py:16: KeyError"}, "steps": [{"name": "Сгенерировать данные для пользователя", "status": "passed", "start": 1719495939630, "stop": 1719495939759}, {"name": "Отправить запрос на создание пользователя", "status": "passed", "parameters": [{"name": "user_body", "value": "{'email': 'leslie76@example.com', 'password': '407055863', 'name': 'austin09'}"}], "start": 1719495939759, "stop": 1719495940271}, {"name": "Отправить запрос на логин пользователя", "status": "passed", "parameters": [{"name": "user_body", "value": "{'email': 'leslie76@example.comSOMETHING_WRONG', 'password': '407055863', 'name': 'austin09'}"}], "start": 1719495940272, "stop": 1719495940641}, {"name": "Отправить запрос на удаление пользователя", "status": "broken", "statusDetails": {"message": "KeyError: 'accessToken'\n", "trace": "  File \"C:\\Users\\ilnazhim\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"D:\\Обучение\\Яндекс практикум\\Diplom\\Diplom_2\\modules\\auth.py\", line 16, in delete_user\n    access_token = user_response.json()[\"accessToken\"]\n                   ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^\n"}, "parameters": [{"name": "user_response", "value": "<Response [403]>"}], "start": 1719495940641, "stop": 1719495940641}], "parameters": [{"name": "field", "value": "'email'"}], "start": 1719495939630, "stop": 1719495940645, "uuid": "cdf776c1-1fcc-4779-be42-cd051958e1a3", "historyId": "54785a2ca7dbbb444f702e7c70494585", "testCaseId": "af6198738ed58d186ebad5a445dfe2c1", "fullName": "tests.test_user_login.TestUserLogin#test_user_login_with_wrong_params", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "9328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}